### PR TITLE
Kalakos Bahrnob Layout Additions

### PR DESCRIPTION
**Pull request by noroadsleft**
## Description

- corrects the position of the rotary encoder
- adds 24 new layouts (I may be *slightly* neurotic...)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by fauxpark — APPROVED** [review]

**Comment by dunk2k** [comment]
Hi @noroadsleft 
As this PR introduces new layouts confirming this board has more than one physical layout, based on PR checklist should the `LAYOUT` layout macro be renamed to `LAYOUT_all`?

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/kalakos/bahrnob/info.json
[the remark this answers is not in the document]
Comment (noroadsleft) [reply keyboards/kalakos/bahrnob/info.json:1]: oop, fixed.

**Comment by noroadsleft** [comment]
> should the `LAYOUT` layout macro be renamed to `LAYOUT_all`?

@dunk2k,

I was struggling to decide whether I should do that in this PR, or issue a second PR targeting `develop`. Looking it over, I can add it here - the key sequence there is already correct.

File: keyboards/kalakos/bahrnob/info.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
                 {"matrix": [0, 13], "x": 13, "y": 0},
                 {"matrix": [1, 14], "x": 14, "y": 0},
 
-                {"matrix": [0, 14], "x": 15.75, "y": 0},
+                {"matrix": [0, 14], "x": 15.5, "y": 0},
```
[the remark this answers is not in the document]
Comment (noroadsleft) [reply keyboards/kalakos/bahrnob/info.json]: Fixed in 96d8daa.

**Review by dunk2k — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**